src/assets/index.css

```diff
@@ -13,3 +13,6 @@
 @import 'css/pagination.css';
 @import 'css/toggle.css';
 @import 'css/video.css';
+
+/* Styling for Resume Builder */
```

Comment:
Please match project conventions and co locate similarly

Reply:
To resolve this I merged all of the styling for each component in a single stylesheet named resumebuilder.css

![Screen Shot 2022-08-19 at 22 28 01](https://user-images.githubusercontent.com/48884692/185727252-8eb11157-ce9f-4690-8adf-43f3e58a3a02.png)